main.js

```diff
@@ -0,0 +1,92 @@
+var inputAmt = document.querySelector('.expense-amount');
+var inputName = document.querySelector('.expense-name');
+var button = document.querySelector('.add-expense');
+var sumSpan = document.querySelector('.expense-total');
+var table = document.querySelector('.expenses');
+var form = document.querySelector('form');
+var sum = [0];
+
+
+// var handleEnter = function(e) {
+//   if (e.keyCode === 13) {
+//     console.log('returnd');
+//     e.preventDefault();
+//   }
+// };
+// inputAmount.addEventListener('keyup', handleEnter);
+// inputName.addEventListener('keyup', handleEnter);
+
+var handleButton = function(e) {
+  // console.log('buttond');
+  e.preventDefault();
+
+  sum.push(Number(inputAmt.value));
+
+  var newRow = document.createElement('tr');
+  table.appendChild(newRow);
+
+  var nameCol = document.createElement('td');
+  nameCol.classList.add('nameTD');
+  nameCol.textContent = inputName.value;
+  newRow.appendChild(nameCol);
+
+  var amtCol = document.createElement('td');
+  amtCol.classList.add('amtTD');
+  amtCol.textContent = inputAmt.value;
+  newRow.appendChild(amtCol);
+
+  var delCol = document.createElement('td');
+  delCol.classList.add('rmv');
+  var delBttn = document.createElement('button');
+  delBttn.textContent = 'delete';
+  delCol.appendChild(delBttn);
+  newRow.appendChild(delCol);
+  inputAmt.value = '';
+  inputName.value = '';
+
+
+};
+button.addEventListener('click', handleButton);
+
+var handleDelete = function(e) {
+  var delRow = event.target.parentNode.parentNode.remove();
+  // delRow.parentNode.remove();
```

Comment:
Careful with commented out or "dead" code. It leads to a "code smell." Just rely on git!